docs/.local_build.sh

```diff
@@ -9,10 +9,10 @@ SCRIPT_DIR="$(cd "$(dirname "$0")"; pwd)"
 cd "${SCRIPT_DIR}"
 
 mkdir -p ../_dist
-cp -r . ../_dist
+rsync -ruv . ../_dist
 cd ../_dist
 poetry run python scripts/model_feat_table.py
-poetry run nbdoc_build --srcdir docs
+poetry run nbdoc_build --srcdir docs --pause 0
```

Comment:
by default, nbdoc pauses for half a second between each doc to "avoid race conditions"

This is unnecessary for us